client/src/components/HeightClipper/HeightClipper.tsx

```diff
@@ -37,7 +40,26 @@ export class HeightClipper extends React.Component<
     };
     this.main = React.createRef();
     this.content = React.createRef();
+
+    this.debounced_mutation_callback = _.debounce(
+      (mutationList: MutationRecord[], observer: MutationObserver) => {
+        this.componentDidUpdate();
```

Comment:
Will look into trimming this down rather than calling the whole componentDidUpdate()